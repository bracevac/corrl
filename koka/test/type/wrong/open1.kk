type open test {
  X
}

type extend test1 {
  Y
}