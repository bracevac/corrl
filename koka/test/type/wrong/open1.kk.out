test/type/wrong/open1.kk(5,13): error: Type test1 is not defined
 hint: bind the variable using forall<test1> ?